documentation/modules/operators/con-operators-namespaces.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// assembly-operators.adoc
+
+[id='con-operators-namespaces-{context}']
+
+= Watching namespaces with Strimzi operators
+
+[role="_abstract"]
+Operators watch and manage Strimzi resources in namespaces.
+The Cluster Operator can watch a single namespace, multiple namespaces, or all namespaces in a Kubernetes cluster.
+The Topic Operator and User Operator can watch a single namespace.
+
+* The Cluster Operator watches for `Kafka` resources
+* The Topic Operator watches for `KafkaTopic` resources
+* The User Operator watches for `KafkaUser` resources 
+
+The Topic Operator and the User Operator can only watch a single Kafka cluster in a namespace.
+And they can only be connected to a single Kafka cluster.  
+
+If a Topic Operator watches more than one namespace, name collisions and topic deletion can occur. This is because each Kafka cluster uses Kafka topics that have the same name (such as `__consumer_offsets`). 
+
+By using a single User Operator with a single namespace, a user with a given username can exist in more than one Kafka cluster.    
+
+If you deploy the Topic Operator and User Operator using the Cluster Operator, they watch the Kafka cluster deployed by the Cluster Operator by default.
+You can also specify a namespace using `watchedNamespace` in the operator configuration.
+
+For a standalone deployment of each operator, you specify a namespace and connection to the Kafka cluster to watch in the configuration.
+
+If you want to use the operators with more than one Kafka cluster, install the clusters to separate namespaces and use a separate operator configured to watch each namespace.
```

Comment:
This is confusing as it suggests the operators can connect to multiple clusters. I would probably remove this.